src/__test__/generateTypeFileContents.test.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+// Test the generate the type file contents
+
+import {getSchemas} from '../cli/config';
+import {generateTypeFileContents, indexPrelude} from '../generateTypeFiles';
+import gql from 'graphql-tag';
+
+const [_, exampleSchema] = getSchemas(__dirname + '/example-schema.graphql');
+
+describe('generateTypeFileContents', () => {
+    it('split types should export response & variables types', () => {
+        const {indexContents, files} = generateTypeFileContents(
+            'hello.js',
+            exampleSchema,
+            gql`
+                query Hello {
```

Comment:
Most of the operation names in webapp start with a lowercase letter.  I assume that that works as expected so I'm sure it's fine to leave this test as is.